edx-solutions/xblock-drag-and-drop-v2
# Improve keyboard and SR instructions

**Description:** This PR improves keyboard and SR instructions across DnDv2.

**JIRA:** [SOL-2157](https://openedx.atlassian.net/browse/SOL-2157)

**Sandbox:**

Currently at 644b6c7a3917fd171f917acb78ed79b5da6daee7
* LMS: http://pr14104.sandbox.opencraft.hosting/
* Studio: http://studio-pr14104.sandbox.opencraft.hosting/

**Testing instructions:**

Verify the following:
1. Draggable items now have SR only span saying "draggable" when laying on the floor and "draggable, grabbed" when someone picked them up (regardless of keyboard or mouse grab).
2. Space and Command+M can no longer be used to pick and drop items + updated help texts.
3. Keyboard help contains SR specific sections to reassure SR users that they can still use DnDv2 + more clear process description for everyone to see and hear.
4. Verbose help messages are removed from items.
5. If no description provided, zones default to "droppable" label.
6. Item bank SR help moved to aria-label attribute.
7. Feedback popup close buttons now read just "Close".

## Reviewers
- [x] OpenCraft: @mtyaka 
- [x] TNL: @cahrens
